### PR TITLE
Add http metrics page

### PR DESCRIPTION
**Pull request by noelledaley**

### CODE REVIEW

File: ui/app/components/http-requests-table.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  classNames: ['http-requests-table'],
+  counters: null,
+  showChangeColumn: computed('counters', function() {
+    const { counters } = this;
+    return counters && counters.length > 1;
+  }),
+  countersWithChange: computed('counters', function() {
+    const { counters } = this;
+
+    if (counters) {
```
Comment (noelledaley) [inline ui/app/components/http-requests-table.js]: is there a better way to ensure this value is only calculated once counters is no longer `null`? it seems odd to have to defensively check if `counters` exists both here and in `showChangeColumn` above.

File: ui/app/components/http-requests-table.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          const change = (((month.total - previousMonthVal) / month.total) * 100).toFixed(1);
+          const newCounter = Object.assign({ change }, month);
+          countersWithPercentChange.push(newCounter);
+        } else {
+          // we're looking at the first counter in the list, so there is no % change value.
+          countersWithPercentChange.push(month);
+          previousMonthVal = month.total;
+        }
+      });
+      return countersWithPercentChange;
```
Comment (noelledaley) [inline ui/app/components/http-requests-table.js]: similarly, does it make sense to store `countersWithPercentChange` as a separate computed property, or should we just update the existing `counters` property inside an `init`?

File: ui/app/components/http-requests-table.js (hunk 1 above)
Comment (meirish) [reply ui/app/components/http-requests-table.js]: The computed property will cache, but will recompute when counters changes value, so you have to account for both cases. You could do `let counters = this.counters || []` above so you could drop this `if` if you wanted.  For the `showChangeColumn`, I'd just move that to the template as `{{#if (gt counters.length 1)}}` (using the helper from here: https://github.com/jmurphyau/ember-truth-helpers#usage).

File: ui/app/components/http-requests-table.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      counters.forEach(month => {
+        if (previousMonthVal) {
+          const percentChange = (((month.total - previousMonthVal) / month.total) * 100).toFixed(1);
+          let glyph;
+          if (percentChange > 0) {
+            glyph = 'arrow-up';
+          } else if (percentChange < 0) {
+            glyph = 'arrow-down';
+          }
+          const newCounter = Object.assign({ percentChange, glyph }, month);
```
Comment (meirish) [inline ui/app/components/http-requests-table.js]: You'll need the ember polyfill here.

File: ui/app/components/http-requests-table.js (hunk 2 above)
Comment (meirish) [reply ui/app/components/http-requests-table.js]: Yeah, it think this separate computed prop is the way to go. If you're mutating the passed counters array, you'd be mutating state on the model as well - you could be copied over, but the computed prop is copy + transform so I think it works.

**Pull request closed without merging**